front/tests/pages/scenario-page-model.ts

```diff
@@ -132,6 +135,11 @@ class PlaywrightScenarioPage {
     await expect(this.getInfraLoadState).toHaveClass(/cached/);
   }
 
+  async set10Trains() {
+    await this.getTrainCountInput.focus();
+    await this.page.keyboard.press('Digit0');
+  }
```

Comment:
```suggestion
    async setNumberOfTrains(digits: string[]) {
       await this.getTrainCountInput.focus();
       await this.page.keyboard.press('Delete');
       digits.forEach((digit) => {
       await this.page.keyboard.press(digit);
       })
  }
```

Reply:
Yes we talked about this, that's a good idea

Reply:
maybe it would be more intuitive for the dev to put a string in function a param and split('') it in the function before forEach ?

Reply:
adding methods like split() makes the code less intuitive I guess, here we can see quickly that we "press" a key with an array, so the iteration is still more intuitive in my opinion